src/meltano/cli/config.py

```diff
@@ -81,6 +84,13 @@ def config(  # noqa: WPS231
         tracker.track_command_event(CliEvent.aborted)
         raise
 
+    if ctx.obj["is_default_environment"]:
+        logger.info(
+            f"Deactivated Environment '{project.active_environment.name}' (the default) as the `meltano config` command does not adhere to your projects `default_environment`. "
```

Comment:
This wording sounds a little weird to me. It makes it seem like the specific value of `default_environment` is incompatible with `meltano config`, while it's just that the setting is ignored for `meltano config`. So how about

```suggestion
            f"The default environment ({project.active_environment.name}) will be ignored for `meltano config`."
```